src/clients/api/queries/getVrtConversionEndTime.ts

```diff
@@ -4,11 +4,13 @@ export interface IGetVrtConversionEndTimeInput {
   vrtConverterContract: VrtConverter;
 }
 
-export type GetVrtConversionEndTimeOutput = string;
+export type GetVrtConversionEndTimeOutput = Date;
 
-const getVrtConverstionEndTime = ({
+const getVrtConverstionEndTime = async ({
   vrtConverterContract,
-}: IGetVrtConversionEndTimeInput): Promise<GetVrtConversionEndTimeOutput> =>
-  vrtConverterContract.methods.conversionEndTime().call();
+}: IGetVrtConversionEndTimeInput): Promise<GetVrtConversionEndTimeOutput> => {
+  const resp = await vrtConverterContract.methods.conversionEndTime().call();
+  return new Date(+resp * 1000);
```

Comment:
It would be nice to add a comment explaining the value retrieved is in seconds and so you convert it into milliseconds.